### ISSUE TITLE
Is it possible to have levels in roles?

### ISSUE THREAD

**Issue by ledzgio** [vintasoftware/django-role-permissions#98]
I am implementing a system where each role (Developer, Influencer, Translator, etc.) can have also levels, so a certain user can be a Level 1 Developer, another user can be Level 2 Translator, and so on. Any idea if it is already possible with this app?

**Comment by filipeximenes** [comment]
@ledzgio hmm good point. If those roles are fixed (ie.: you always know how many levels there are), you can just have a Role class for each. But they will be isolated, you'll have to provision them as if they were completely unlinked.

**Comment by ledzgio** [comment]
@filipeximenes yes levels are fixed in my idea, means I should create classes like Developer Level 1, Developer Level 2 and so forth, right?

**Comment by filipeximenes** [comment]
@ledzgio that's right!
